app/app_test.go

```diff
@@ -40,3 +65,134 @@ func TestGetMaccPerms(t *testing.T) {
 	dup := GetMaccPerms()
 	require.Equal(t, maccPerms, dup, "duplicated module account permissions differed from actual module account permissions")
 }
+
+func TestRunMigrations(t *testing.T) {
```

Comment:
If this is already testing in the sdk, do we need to test in regen ledger? Are there any significant differences?

Reply:
Yeah, not required.